dev/ToastNotifications/ToastNotificationManager.cpp

```diff
@@ -96,7 +96,11 @@ namespace winrt::Microsoft::Windows::ToastNotifications::implementation
     }
     winrt::Microsoft::Windows::ToastNotifications::ToastNotificationSetting ToastNotificationManager::Setting()
     {
-        throw hresult_not_implemented();
+        std::wstring appId { RetrieveAppId() };
+        DWORD toastNotificationSetting{ 0 };
+        ToastNotifications_QuerySettings(appId.c_str(), 0 /* ToastNotifications */, &toastNotificationSetting);
```

Comment:
@sharath2727 Please get rid of the toastNotificationtype argument (second one) in the UDK. It is redundant.

Reply:
This will be a different task to fix the UDK correct?